index.bs

```diff
@@ -76,773 +93,795 @@ All <{iframe}> use cases are still valid and should continue to use <{iframe}>,
 
-*This section is non-normative.*
-
-<!-- ====================================================================== -->
-## Latency-sensitive applications in virtualized sessions ## {#browser-content-redirection}
-<!-- ====================================================================== -->
+For convenience, the Controlled Frame specification assumes that the Fenced
+Frame specification is in place. There are concepts introduced in the Fenced
+Frame specification, such as top-level traversibles, that are broadly useful
+to refer to in the context of Controlled Frame.
 
-In virtualized environments, users typically have a local thin client that
-renders a full virtual desktop. The actual desktop execution environment will be
-running on a remote virtualization server. If the user's browser navigates to a
-latency-sensitive application (such as a video app), the rendered content will
-have additional latency ("lag") that makes the experience difficult or
-impossible for the user. This also applies for applications that record the
-user, such as video conferencing applications. In these latency-sensitive
-applications, the virtual desktop application can render the latency-sensitive
-content locally and overlay it on top of the rendered remote content to reduce
-this latency. This use case is also known as "browser content redirection."
+The Fenced Frame specification achieves defining these concepts via monkey
+patching some specifications, such as HTML. We will also require monkey
+patching specifications for some parts of this Controlled Frame specification.
 
 <!-- ====================================================================== -->
-## Embedding third party web content without restriction ## {#no-embedding-prohibitions}
+<h2 id=the-controlledframe-element>The <dfn element export>controlledframe</dfn> element</h2>
 <!-- ====================================================================== -->
 
-In a kiosk environment, applications must load content from third parties and
-display that content on screens within their applications. A teacher may trigger
-the navigation event, or it may be configured by an administrator such as a
-shopping mall manager. The content may prohibit embedding by <{iframe}> through
-the use of X-Frame-Options and CSP. An controlled frame, however, should be able
-to load all content, even content that prohibits embedding by <{iframe}>.
+<dl class="element">
+ <dt>[=Contexts in which this element can be used=]:</dt>
+ <dd>Where [=embedded content=] is expected.</dd>
+ <dt>[=Content model=]:</dt>
+ <dd>[=Nothing=].</dd>
+ <dt>[=Content attributes=]:</dt>
+ <dd>[=Global attributes=]</dd>
+ <dd><code>[=width=]</code> — Horizontal dimension</dd>
+ <dd><code>[=height=]</code> — Vertical dimension</dd>
+ <dt>[=Accessibility considerations=]:</dt>
+ <dd><p class=XXX>TODO</p></dd>
+ <dt>[=DOM interface=]:</dt>
+ <dd>
+<xmp class=idl>
+[Exposed=Window, SecureContext]
```

Comment:
Can we replace this with IsolatedContext? I didn't do that with WebRequest but probably should have.

Reply:
Done.